modules/create-object-collections.adoc

```diff
@@ -0,0 +1,66 @@
+// Module included in the following assemblies:
+//
+// * operating/create-use-collections.adoc
+:_content-type: PROCEDURE
+[id="creating-object-collections_{context}"]
+= Creating object collections
+
+To access collections, click *Platform Configuration* -> *Collections*. The page displays a list of currently-configured collections. You can perform the following actions:
+
+* Search for collections by entering text in the *Search by name* field, and then press *->*. 
+* Click on a collection in the collection list to view the collection in read-only mode.
+* Click {kebab} for an existing collection to edit, clone, or delete it.
++
+[NOTE]
+====
+You cannot delete a collection that is actively used in {product-title-short}. ##I tried testing this, but I didn't get an error message and it allowed me to delete a collection that I'd used in a vulnerability report. Using stagingdb.demo.stackrox.com##
```

Comment:
When did you test this? I just checked and it correctly error'd for me on stagingdb.

It looks like the code that ensures that you can't delete an in use collection was merged this Tuesday and available on staging Tuesday at [3:04PM (EST)](https://srox.slack.com/archives/C3W3U0RGW/p1675195482160129). If you tested after that point we should re-investigate as that could be a possible bug.

Reply:
Thanks, it might have just not been merged when I tested. I do see an error message now - thank you!